src/audio/kpb.c

```diff
@@ -356,57 +375,54 @@ static int kpb_copy(struct comp_dev *dev)
  */
 static void kpb_buffer_data(struct comp_data *kpb, struct comp_buffer *source)
 {
-	trace_kpb("kpb_buffer_data()");
 	int size_to_copy = kpb->source_period_bytes;
```

Comment:
@mrajwa source_period_bytes doesnt seem to be set at all. Was the intention to use source->avail instead?

Reply:
@ranj063 yes, KPB relies only on sink->free / source->avail now.